moveit_core/collision_detection_bullet/src/collision_env_bullet.cpp

```diff
@@ -34,13 +34,15 @@
 
 /* Author: Jens Petit */
 
-#include <moveit/collision_detection_bullet/collision_env_bullet.h>
-#include <moveit/collision_detection_bullet/collision_detector_allocator_bullet.h>
-#include <moveit/collision_detection_bullet/bullet_integration/ros_bullet_utils.h>
-#include <moveit/collision_detection_bullet/bullet_integration/contact_checker_common.h>
 #include <boost/bind.hpp>
+
 #include <bullet/btBulletCollisionCommon.h>
```

Comment:
or, we just update all bullet includes to use the `bullet/` prefix. Afaik the bullet tutorials use includes without prefix though.